openedx/features/wikimedia_features/wikimedia_general/templates/wikimedia_general/edx_ace/threadcreationnotification/email/body.txt

```diff
@@ -2,6 +2,10 @@
 {% block content %}
     {% blocktrans trimmed %}{{ thread_username }} created a post: {{ thread_title }}:{% endblocktrans %}
     {{ thread_body }}
+    {% if courseware_url %}
     {% trans "View discussion" %} -> {{ post_link }}
+    {% else %}
+    {% trans "View discussion" %} -> {{ courseware_url }}
+    {% endif %}
```

Comment:
how about sending redirect link in general variable named `discussion_url` instead of these two different ones. This way we could avoid this decision here without hurting readability.

Reply:
I have kept it as post_link for reusability of code purpose, but not have used this condition in case if there is courseware_url then post_link contains it else it points to general discussion section